lib/spack/llnl/util/filesystem.py

```diff
@@ -569,6 +569,9 @@ def _find_recursive(root, search_files):
     # found in a key, and reconstructing the stable order later.
     found_files = collections.defaultdict(list)
 
+    if not os.path.isdir(root):
```

Comment:
i guess you can now drop changes in `filesystem`?

Reply:
Will do. One sec.